test/end-to-end/cases/focus-vs-pending.xspec

```diff
@@ -54,22 +54,38 @@
       </t:pending>
 
       <!--
-         Test @pending.
+         Test x:scenario/@pending.
       -->
       <t:scenario label="an erroneous scenario with @pending must be Pending"
-                  pending="testing @pending without @focus">
+                  pending="testing x:scenario[empty(@focus)]/@pending">
          <t:expect label="it would throw an error if it were not Pending"
                    test="error()" />
       </t:scenario>
 
       <!-- does not make sense, really, but to see how the implem reacts -->
       <t:scenario label="a Success scenario with both @pending and @focus (not recommended as ambiguous)"
-                  pending="testing @pending with @focus"
-                  focus="testing @focus with @pending">
+                  pending="testing x:scenario[@focus]/@pending"
+                  focus="testing x:scenario[@pending]/@focus">
          <t:expect label="must execute the test and return Success"
                    select="9" />
       </t:scenario>
 
+      <!--
+         Test x:expect/@pending.
+      -->
+      <t:scenario label="a Success x:expect with @pending must be Pending">
+         <t:expect label="it would return Success if it were not Pending"
+                   select="9"
+                   pending="testing x:expect/@pending without ancestor::x:scenario/@focus" />
+      </t:scenario>
+
+      <t:scenario label="a Success x:expect with both @pending and ancestor::x:scenario/@focus (not recommended as ambiguous)"
+                  focus="testing x:scenario[@focus] with child::x:expect/@pending">
+         <t:expect label="must execute the test and return Success"
+                   select="9"
+                   pending="testing x:expect/@pending with ancestor::x:scenario/@focus" />
+      </t:scenario>
```

Comment:
This structure makes a lot more sense than the situation on line 66. I think this scenario's behavior (along with the analogous case where you mark an assertion as pending via parent `t:pending` element rather than `@pending`) should be up for reconsideration during work on #575. For now, capturing the situation and recording the current behavior are good.

Tangentially related: It is curious that adding `@focus` to the only scenario element in `test/issue-446_4.xspec` changes the behavior.

Reply:
Thanks @galtm . Good point. Noted it in #575. 
The more I look into the current behavior, I'm inclined to stop propagating `@focus`'s effect at its descendant `x:pending` and `@pending`.